modules/data.atmosphere/R/download.raw.met.module.R

```diff
@@ -1,3 +1,30 @@
+#' @name download.raw.met.module
+#' @title download.raw.met.module
+#'
+#' @param dir directory to write outputs to
+#' @param met source included in input_met
+#' @param register register.xml, provided by met.process
+#' @param machine machine associated with hostname, provided by met.process
+#' @param start_date the start date of the data to be downloaded (will only use the year part of the date)
+#' @param end_date the end date of the data to be downloaded (will only use the year part of the date)
+#' @param str_ns substitute for site_id if not provided, provided by met.process
+#' @param con database connection based on dbparms in met.process
+#' @param input_met Which data source to process
+#' @param site.id site id
+#' @param lat.in site latitude, provided by met.process
+#' @param lon.in site longitude, provided by met.process
+#' @param host host info from settings file
+#' @param site site info from settings file
+#' @param username database username
+#' @param overwrite whether to force download.raw.met.module to proceed
+#' @param dbparms database settings from settings file
+#' @param Ens.Flag 
+#'
+#' @return
+#' @export
+#'
+#' @examples
+
 .download.raw.met.module <-
```

Comment:
I usually expect function names starting with dots or underscores to be for internal use only. Are we sure we want to export this?

Reply:
Adding the @export to this function makes it easier to run met.process outside of do_conversions.